docs/data/charts/tooltip/tooltip.md

```diff
@@ -7,8 +7,66 @@ title: Charts - Tooltip
 
 <p class="description">Tooltip provides extra data on charts item.</p>
 
+In all charts components, you can pass props to the tooltip by using `tooltip={{...}}`.
+If you are using composition, you can add the `<Tooltip />` component and pass props directly.
+
 ## Interactions
 
+The tooltip can be triggered by two kind of events:
+
+- `'item'` which means user mouse is hover an item of the chart. The tooltip will display data about this specific items.
+- `'axis'` which means user mouse position is associated to a value of the x-axis. The tooltip will display data about all series at this specific x value.
+
+{{"demo": "Interaction.js", "bg": "inline"}}
+
 ## Highlights
 
-## Rendered Tooltip
+## Highlighting axis
+
+You can hightligh data based on mouse position.
+By default those highlights are lines, but is can also be a vertical band if your x-axis use `scaleType: 'band`.
+
+On chart, to customize this behavior,you can use
+
+```jsx
+highlight={{
+    x: 'line', // Or 'none', or 'band'
+    y: 'line', // Or 'none'
+    }}
+```
+
+{{"demo": "BandHighlight.js",  "bg": "inline"}}
+
+## Highlighting series 🚧
```

Comment:
I find this demo with tons of circles very weird
If we want to keep the circles visible, I would be in favor of drastically reducing the amount of data in each serie